docs/roadmap.md

```diff
@@ -19,12 +19,33 @@ The following roadmap gives an overview of what we are currently working on and
 
 *Status*: tech spec out
 
-#### Integrate with data quality system
+#### Integrate with Data Quality system
 *What*: Integrate with different data quality systems to provide quality score.
 
 *Status*: planning
 
 ## Mid Term - Our Next steps
+#### Improve search ranking
+*What*: Update search ranking to be informed by "badges" that may exist on data sets e.g. deprecated, etc.
+
+*Status*: planning
```

Comment:
this one is committed for discovery and explore work.

Reply:
If it's that close, then I may just take this out of the roadmap. In general, I do think there's a lot of scope for search improvements but it's not quite clear to me what we'd have to do for that.

Thoughts?

Reply:
agree, I think we haven't yet defined a concrete search pain point story which I recalled @danwom will do some user study as well?